tools/patches/021-add-navbar-bg-vars.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/inst/lib/bs/scss/_variables.scss b/inst/lib/bs/scss/_variables.scss
-index 7a42c44..aee26ce 100644
+index af39c76..e332bbc 100644
 --- a/inst/lib/bs/scss/_variables.scss
 +++ b/inst/lib/bs/scss/_variables.scss
-@@ -747,6 +747,9 @@ $navbar-toggler-padding-x:          .75rem !default;
- $navbar-toggler-font-size:          $font-size-lg !default;
- $navbar-toggler-border-radius:      $btn-border-radius !default;
+@@ -749,6 +749,9 @@ $navbar-toggler-border-radius:      $btn-border-radius !default;
+ 
+ $navbar-nav-scroll-max-height:      75vh !default;
```

Comment:
Added in https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0#diff-108a57a50d5ebe203f447ef7bf1e5e53826bdb82baab2baa2b41b4158cdd4d09R734-R735